ppr-ui/src/components/common/ContactInformation.vue

```diff
@@ -270,7 +272,7 @@ export default defineComponent({
     })
 
     watch(() => localState.contactInfoModel, (val) => {
-      props.setStoreProperty(val)
+      emitStoreUpdate(val)
```

Comment:
Switched from calling the passed setter to emitting store updates up to the parent.

Reply:
Sounds good! 
No impact on previous PPR implementations?

Reply:
This is a new component and so far used only in UnitNotes.